css/style.css

```diff
@@ -0,0 +1,230 @@
+/* Busra's Cake Company */
```

Comment:
Consider using normalize.css styles for your projects (as in previous week exercise) - this is a good practice to use.